tests/evaluation_metrics/test_precision_recall_records.py

```diff
@@ -0,0 +1,22 @@
+from pytest import approx
+
+from datasetinsights.evaluation_metrics import PrecisionRecallRecords
+
+
+def test_precision_recall_records(get_mini_batches):
+    mini_batches = get_mini_batches
+
+    pr_records = PrecisionRecallRecords()
+    expected_precision = [1, 0.5, 0.33333, 0.5]
+    expected_recall = [0.33333, 0.33333, 0.33333, 0.66667]
+    for mini_batch in mini_batches:
+        pr_records.update(mini_batch)
```

Comment:
Is it possible to decouple update and compute in two unit test case? it seems alot of assertion in one place.

Reply:
Update and compute happen in series. I think it's good to combine them together. So as other evaluation metrics unit tests, like `test_average_precision_2d.py`.